cql3/statements/select_statement.cc

```diff
@@ -881,7 +883,10 @@ lw_shared_ptr<const service::pager::paging_state> indexed_table_select_statement
     if (!results->row_count() || *results->row_count() == 0) {
         return std::move(paging_state);
     }
-    auto [last_base_pk, last_base_ck] = result_view.get_last_partition_and_clustering_key();
+
+    auto&& last_partition_and_clustering_key = result_view.get_last_partition_and_clustering_key();
+    auto& last_base_pk = std::get<0>(last_partition_and_clustering_key);
+    auto& last_base_ck = std::get<1>(last_partition_and_clustering_key);
```

Comment:
Is the change of "auto" before the patch to "auto&" after the patch, necessary? I don't know how to review it before going back to look at the entire code surrounding it.

Reply:
It's not necessary. As far as I can tell there is no difference in behavior. `auto&&` resolves to `auto`.